google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -345,7 +346,11 @@ public static QueryParameterValue interval(String value) {
     return of(value, StandardSQLTypeName.INTERVAL);
   }
 
-  /** Creates a {@code QueryParameterValue} object with a type of INTERVAL. */
+  /**
+   * Creates a {@code QueryParameterValue} object with a type of INTERVAL. This method is obsolete.
+   * Use {@link #interval(String)} instead.
+   */
+  @ObsoleteApi("Use interval(String) instead")
   public static QueryParameterValue interval(PeriodDuration value) {
```

Comment:
@lqiu96 I'm not sure if this makes sense. [`PeriodDuration`](https://www.threeten.org/threeten-extra/apidocs/org.threeten.extra/org/threeten/extra/PeriodDuration.html) is part of Threeten Extra which only accepts `java.time` objects.
I'm guessing we may want to keep it since it's just a helper class that _happens_ to be from ThreeTen but not meant to be a backport (since this is not available in the JDK).

Reply:
Hmm. Ideally, I think we should try to migrate away from threeten-extra as well. I'm guessing from your comment that PeriodDuration doesn't exist in the JDK so we can't build a 1:1 method signature replacement with the available JDK alternative.

If `interval(String)` has the same behavior, then I think that probably can use that as the alternative. But would be a question for the Bigquery SMEs to confirm that they have the same behavior.

Reply:
Thank you. @PhongChuong can you please confirm whether marking this `threeten-extra`-based method as `@ObsoleteApi` makes sense? And does `interval(String)` as the suggested alternative make sense?

Reply:
@diegomarquezp , marking it as @ObsoleteApi and the suggested alternative makes sense.

Reply:
Thanks!